src/base/extensions.ts

```diff
@@ -39,3 +43,28 @@ export interface IStartMenuSection {
 
   update?(): void;
 }
+
+
+export const EXTENSION_POINT_STARTMENU_DATASET = 'ordinoStartMenuDataset';
+
+export interface IStartMenuDatasetDesc extends IPluginDesc {
+  id: string;
+  name: string;
+  headerIcon: string;
+  viewId: string;
+  datasource: any;
```

Comment:
`datasource` seems to be specific to tdp_publicdb. So [my comment in your other PR](https://github.com/Caleydo/tdp_publicdb/pull/132#discussion_r591862151).

Furthermore, please rename `datasource` to `dataSource` in your code, since the origin is "data source".